v1/pkg/client/api/openapi.yaml

```diff
@@ -2695,19 +2695,23 @@ components:
     AvailableResources:
       example:
         Networks:
-        - IPPoolID: IPPoolID
+        - VNI: 6
+          IPPoolID: IPPoolID
           Description: Private network for intra-cluster communication
           LocationID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
           HostUse: Default
           Purpose: Storage
           ID: 046b6c7f-0b8a-43b9-b35d-6489e6daee91
+          VLAN: 0
```

Comment:
This is definitely not right. VNI may be 0 for non-EVPN network types, but VLAN ID will always be non-zero. Would be better to show this VLAN with example values like 200.